lib/utils.js

```diff
@@ -56,8 +56,9 @@ var parseIntArray = function(arrayOfInts) {
 
 var momentRange = function(startDate, endDate, timeGranularity) {
   timeGranularity = constants.timeGranularities[timeGranularity];
-  startDate = moment.utc(startDate);
-  endDate = moment.utc(endDate);
+  // -- problem with meteor and moment, invalid moment objects
+  startDate = moment.utc(startDate.toISOString());
+  endDate = moment.utc(endDate.toISOString());
```

Comment:
What I think is needed is probably a little utility function that can check whether `startDate` and `endDate` are a moment object or not. For example:
```
var toMoment = function(date) {
  if (date instanceof moment) return moment.utc(date.format());
  return moment.utc(date);
};

// Usage
var momentRange = function(...) {
  // ...
  startDate = toMoment(startDate);
  // etc.
}
```
Something like that might also work for your incompatibility problem. What do you think?